cmd/kubeadm/app/util/version_test.go

```diff
@@ -72,7 +75,10 @@ func TestInvalidVersion(t *testing.T) {
 	}
 	for _, s := range invalidVersions {
 		t.Run(s, func(t *testing.T) {
-			ver, err := KubernetesReleaseVersion(s)
+			fileFetcher := func(url string, timeout time.Duration) (string, error) {
```

Comment:
You can reuse the same func from `TestValidVersion`. Probably extract it in the file and give it a name.